IceNet/polar-modelling-icenet.ipynb

```diff
@@ -0,0 +1,1411 @@
+{
```

Comment:
<p>"same input data of Nature Communications' paper " =&gt; "same input data as in the  Nature Communications paper "</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/PangeoExamples/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>done!</p>